app/components/blacklight/top_navbar_component.html.erb

```diff
@@ -0,0 +1,192 @@
+<nav id="skip-link" role="navigation" aria-label="Skip links">
```

Comment:
In BL, the skip-link is on the base template. https://github.com/projectblacklight/blacklight/blob/abf6f24bb8e53354dc91db8520f029bee68c4a4c/app/views/layouts/blacklight/base.html.erb#L31-L37. Can we continue to follow that paradigm here?  That makes it easier for people who navigate between the two code bases.

Reply:
Done. Thanks.